libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Preds/CertState.hs

```diff
@@ -240,6 +245,16 @@ certStatePreds _p =
     gdKeyHashSet = var "gdKeyHashSet" (SetR GenDelegPairR)
     gdKeyHashList = var "gdKeyHashList" (ListR GenDelegPairR)
 
+certStateCheckPreds :: Era era => Proof era -> [Pred era]
+certStateCheckPreds p =
+  [ NotMember (Lit CoinR (Coin 0)) (Rng stakeDeposits)
+  , Dom rewards :=: Dom stakeDeposits
+  , Dom delegations :⊆: Dom rewards
+  , case p of
+      Conway {} -> Sized (ExactSize 0) ptrs
+      _ -> Dom rewards :=: Rng ptrs
```

Comment:
I think this should be some thing like this
```
if protocolVersion p >= protocolVersion (Conway Standard)
   then  Sized (ExactSize 0) ptrs
   else Dom rewards :=: Rng ptrs
```
This way it will hold in all Eras greater than Conway
By the way one can find protocolVersion   by  import  Test.Cardano.Ledger.Generic.Functions(protocolVersion )

Reply:
Thanks for the pointer!